packages/component/src/BasicWebChat.js

```diff
@@ -82,9 +85,11 @@ const BasicWebChat = ({ className }) => {
 export default BasicWebChat;
 
 BasicWebChat.defaultProps = {
-  className: ''
+  className: '',
+  sendBoxRenderer: undefined
```

Comment:
Rather than renderer, this will need to be changed to `sendBoxMiddleware`.
![image](https://user-images.githubusercontent.com/14900841/95793093-af915900-0c99-11eb-8e8d-efe1da383924.png)
https://github.com/microsoft/BotFramework-WebChat/tree/master/packages/component/src/Middleware

I added a new issue to get rid of our deprecations to reduce confusion: https://github.com/microsoft/BotFramework-WebChat/issues/3534

@compulim, do you have thoughts?

Reply:
I've changed the name of the prop.

"Middleware" implies that can it can be inserted in between existing logic.
This prop is very much an alternative route that can be used, once used it can't pass on to existing logic.

Having said that, do you still feel this is the correct name for the prop?
Or did you mean my solution should be a middleware and I have to change my implementation?

Reply:
The implementation should be middleware. :) I think there are other considerations that @compulim would like to comment on as well. Pinging him.

Reply:
@compulim is the change satisfactory?